.github/workflows/release.yml

```diff
@@ -9,7 +9,33 @@ on:
       - "[0-9]+.[0-9]+.[0-9]+"
 
 jobs:
+  # We need to verify the version in some packages with the tag in order to have the same version in all packages (python, csharp, angular, web).
```

Comment:
@dbrasseur-aneo can you check this comment? If it's good, it will be merged.

Reply:
Looks good to me